crates/sncast/src/helpers/interactive.rs

```diff
@@ -1,64 +1,69 @@
 use crate::helpers::config::get_global_config_path;
-use anyhow::{anyhow, Context, Result};
+use anyhow::{Context, Result};
 use camino::Utf8PathBuf;
 use configuration::search_config_upwards_relative_to;
 use dialoguer::theme::ColorfulTheme;
 use dialoguer::Select;
 use std::env::current_dir;
+use std::fmt::Display;
 use std::fs;
 use toml_edit::{DocumentMut, Item, Table, Value};
 
+enum PromptSelection {
+    GlobalDefault(String, Utf8PathBuf),
+    LocalDefault(String, Utf8PathBuf),
```

Comment:
```suggestion
    GlobalDefault(Utf8PathBuf),
    LocalDefault(Utf8PathBuf),
```
These are fixed messages anyway, create them in `Display` impl.